pkg/metrics/neutron.go

```diff
@@ -14,6 +14,9 @@ var (
 	neutronFloatingIPCreated   *prometheus.GaugeVec
 	neutronFloatingIPUpdatedAt *prometheus.GaugeVec
 
+	// Status from https://docs.openstack.org/api-ref/network/v2/index.html?expanded=show-floating-ip-details-detail#show-floating-ip-details
+	floatingIpStatus = []string{"ACTIVE", "DOWN", "ERROR"}
+
 	floatingIPLabels = []string{"id", "floating_ip", "fixed_ip", "port_id"}
```

Comment:
```suggestion
	floatingIPLabels = []string{"id", "floating_ip", "fixed_ip", "port_id", "status"}
```

You can add the label here as well and remove the `append` in L.29